kpi_dashboard_altair/static/src/js/altair_widget.js

```diff
@@ -0,0 +1,29 @@
+odoo.define('kpi_dashboard.AltairWidget', function (require) {
+    "use strict";
+
+    var AbstractWidget = require('kpi_dashboard.AbstractWidget');
+    var registry = require('kpi_dashboard.widget_registry');
+
+    var AltairWidget = AbstractWidget.extend({
+        template: 'kpi_dashboard.altair',
+        fillWidget: function (values) {
+            var widget = this.$el.find('[data-bind="value"]');
+            widget.css('height', this.widget_size_y - 90);
+            vegaEmbed(
+                widget[0],
+                values.value.altair,
+                this.altairOptions(values)
+            );
+        },
+        altairOptions: function () {
```

Comment:
When I try, here is the result I get

![2020-04-29_10-15](https://user-images.githubusercontent.com/1853434/80574619-7b372280-8a02-11ea-9c8f-6901177ee984.png)

I have no skill in .js , sorry, but may be it's a bad config on my side ?

Reply:
My fault!!! gitignore is ignoring the files under the folder lib. I just need to include them manually :wink: